universal-application-tool-0.0.1/app/services/program/ProgramBlockDefinitionNotFoundException.java

```diff
@@ -1,5 +1,9 @@
 package services.program;
 
+/**
+ * ProgramBlockDefinitionNotFoundException is thrown when the block definition is not found by the
+ * specified ID.
```

Comment:
suggestion (blocking): the block with the given ID may exist; it just doesn't exist in this program

```suggestion
 * ProgramBlockDefinitionNotFoundException is thrown when the specified block definition is not found in this program.
```

Reply:
Thanks for clarifying suggestion!